pkg/minikube/config/types.go

```diff
@@ -43,6 +43,7 @@ type MachineConfig struct {
 	InsecureRegistry    []string
 	RegistryMirror      []string
 	HostOnlyCIDR        string // Only used by the virtualbox driver
+	ServiceCIDR         string // Use for insecure registry
```

Comment:
I don't understand why the ServiceCIDR is only to be used with the insecure registry. Can you clarify this comment?